site/es/blog/new-in-devtools-98/index.md

```diff
@@ -30,202 +28,296 @@ draft: true
+
 <!-- Previously, the accessibility tree was available in the **Accessibility** pane. The view is limited, it only enables you to explore a single node and its ancestors. -->
 
+Anteriormente, el árbol de accesibilidad estaba disponible en el panel de **Accesibilidad**. La vista es limitada, solo le permite explorar un único nodo y sus ancestros.
+
 <!-- Our team is still actively working on this preview feature. We are looking for your [feedback](https://goo.gle/devtools-a11y-tree-feedback) for further enhancements! -->
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/o4BY07JabERFd6OieU8b.png", alt="Full-page accessibility tree", width="800", height="505" %}
+Nuestro equipo está aún trabajando activamente en esta novedad. ¡Estamos buscando tu [feedback](https://goo.gle/devtools-a11y-tree-feedback) para futuras mejoras!
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/o4BY07JabERFd6OieU8b.png", alt="Árbol de accesibilidad de pantalla completa", width="800", height="505" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/847a06a6535111826f898175b210dbe0948277a0 #}
 
 Chromium issue: [887173](https://crbug.com/887173)
 
-
 <!-- ## More precise changes in the Changes tab {: #changes }  -->
 
+## Cambios más precisos en la pestaña Cambios {: #changes }
+
 <!-- The code changes in the **Changes** tab is pretty-printed automatically.  -->
 
+Los cambios de código en la pestaña **Cambios** se imprimen desplegados automáticamente.
+
 <!-- Previously, it was hard to trace the actual changes of minified source code because all the code is shown in a single line.  -->
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/aup2bT490dkvuBu3o4DS.png", alt="Changes tab", width="800", height="450" %}
+Anteriormente, era difícil rastrear los cambios reales del código fuente minificado porque el código se muestra en una sola línea.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/aup2bT490dkvuBu3o4DS.png", alt="Pestaña Cambios", width="800", height="450" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/4382b533525c65fbdb1785eda2babf035ad8bcb8 #}
 
 Chromium issues: [1238818](https://crbug.com/1238818), [1268754](https://crbug.com/1268754) , [1086491](https://crbug.com/1086491)
 
-
 <!-- ## Set longer timeout for user flow recording {: #recorder-timeout } -->
 
+## Establecer un tiempo de espera más largo para la grabación del flujo de usuario {: #recorder-timeout }
+
 <!-- You can now adjust the **Timeout** settings in the [Recorder](/docs/devtools/recorder/) for all steps or a specific step. This is useful especially for pages with slow network requests and lengthy animation. -->
 
+Ahora puede ajustar la configuración de **Tiempo de espera** (_timeout_) en la [Grabadora](/docs/devtools/recorder/) para todos los pasos o para un paso específico. Esto es útil especialmente para páginas con solicitudes de red lentas y animación prolongada.
```

Comment:
The strings of the Recorder panel don't have a translation yet. As such, I think we should leave the strings of the panel we mention here in English, so that the terms we refer to are the same as ones the user will see in DevTools. Could we please leave these strings as is?
```suggestion
Ahora puede ajustar la configuración de **Tiempo de espera** (_timeout_) en el panel [Recorder](/docs/devtools/recorder/) para todos los pasos o para un paso específico. Esto es útil especialmente para páginas con solicitudes de red lentas y animación prolongada.
```